src/utils/crypto/index.ts

```diff
@@ -0,0 +1,31 @@
+import { KEK } from 'pages/generate';
+import { B64EncryptionResult } from 'services/uploadService';
+import { KeyAttributes } from 'types';
+import CryptoWorker from './cryptoWorker';
+
+export async function generateIntermediateKey(
```

Comment:
`generateIntermediateKeyAttributes` since it's returning all the attributes and not just the `intermediateKey`??